src/Uno.UI/UI/Xaml/Media/AcrylicBrush/AcrylicBrush.wasm.cs

```diff
@@ -53,5 +53,15 @@ internal static void ResetStyle(UIElement element)
 		/// <returns>Value indicating whether backdrop filter is supported.</returns>
 		private static bool IsBackdropFilterSupported() =>
 			_isBackdropFilterSupported ??= WindowManagerInterop.IsCssFeatureSupported(CssSupportCondition);
+
+		protected override void OnConnected()
+		{
+			base.OnConnected();
+		}
+
+		protected override void OnDisconnected()
+		{
+			base.OnDisconnected();
+		}
```

Comment:
Does not seem needed.

Reply:
I think it was needed due to the reference implementation comparer

Reply:
Yes. If `Skia` overrides these, then both Reference and Wasm need to.

The reason is if user is doing:

```csharp
public class MyAcrylicBrush : AcrylicBrush
{
    protected override void OnConnected() {/*some code*/ base.OnConnected(); }
}
```

We want `base.OnConnected` to call into `AcrylicBrush.OnConnected` on Skia.

But, at compile time, we only have Reference binaries. So, we have to keep `OnConnected` in Reference. And, if it exists in Reference, it must also exist in Wasm to avoid MissingMethodException at runtime